### DISCUSSION TITLE
Cannot create or edit a record through the admin UI

### DISCUSSION THREAD

**Discussion by BPilot253** [pocketbase/pocketbase#1245] [Q&A] [answered]
*Another* issue. When I try to create or edit a record through the admin UI I get "Failed to load the submitted data due to invalid formatting." error and in the terminal it shows `multipart: NextPart: EOF`.

When creating or editing a record using the JavaScript SDK, this works correctly.

What's the issue here?

**Comment by ganigeorgiev** [comment, marked as answer]
I guess when using the SDK you are passing an object (aka. sending JSON as request body), but the record creation in the Admin UI is using miltipart/form-data (file upload is supported only with multipart/form-data).

Since you are using a proxy (https://github.com/pocketbase/pocketbase/discussions/1239), make sure that the correct headers (eg. `Content-Type`) and body are proxied back to the PocketBase server.

**Reply by ganigeorgiev** [reply]
If you are using `express-http-proxy`, then it could be also possible that the body size limit is reached and you can try to increase it - https://github.com/villadora/express-http-proxy#limit

**Reply by ganigeorgiev** [reply]
You can also check https://github.com/villadora/express-http-proxy/issues/127

**Reply by BPilot253** [reply]
The issue was just a big stupidity from my side. I forgot I had body parser on the whole app. I removed it and now it works like a charm. Thanks for taking time to help me...
